packages/vscode-lit-plugin/esbuild.script.mjs

```diff
@@ -0,0 +1,33 @@
+import * as esbuild from "esbuild";
+
+const buildUs = esbuild.build({
+  entryPoints: ["src/extension.ts"],
+  bundle: true,
+  outfile: "built/bundle.js",
+  platform: "node",
+  minify: true,
+  target: "es2017",
+  format: "cjs",
+  color: true,
+  external: ["vscode", "typescript"],
+  mainFields: ["module", "main"]
+});
+
+const buildTsLitPlugin = esbuild.build({
+  entryPoints: ["../ts-lit-plugin/src/index.ts"],
+  bundle: true,
+  outfile: "built/node_modules/ts-lit-plugin/lib/index.js",
+  platform: "node",
+  external: ["typescript"],
+  minify: true,
+  target: "es2017",
+  format: "cjs",
+  color: true,
+  mainFields: ["module", "main"]
+});
+
+try {
+  await Promise.all([buildUs, buildTsLitPlugin]);
+} catch (e) {
+  process.exit(1);
```

Comment:
Log the error?

Reply:
This is the recommended style from the esbuild docs, the error has some useful info but it's mostly a retread of the error that's already been logged at that point.

On the other hand... simplicity is nice, and this is hella fast already. Switched to just awaiting directly.